cluster/addons/cluster-loadbalancing/glbc/default-svc-controller.yaml

```diff
@@ -1,33 +1,26 @@
-{% set kube_uid = "" -%}
-{% if pillar['kube_uid'] is defined -%}
-  {% set kube_uid = pillar['kube_uid'] %}
```

Comment:
If gke keeps re-salting clusters it defeats the purpose of file based persistence, and there are too many edge cases anway, so it's safer to maintain and allocate the uid in the l7 pod.